.github/workflows/depsreview.yaml

```diff
@@ -0,0 +1,14 @@
+name: 'Dependency Review'
+on: [pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repository'
+        uses: actions/checkout@v3
```

Comment:
Sorry, but wasn't it you who claimed that using versions is very insecure? ;)
https://github.com/google/syzkaller/commit/3e71f314c953e95ab6d9ff5c4da398ca6ef492c5

Reply:
👍  I agree. Fixed it. :)